subprojects/docs/src/samples/userguide/tutorial/osgi/kotlin/build.gradle.kts

```diff
@@ -0,0 +1,26 @@
+plugins {
+  osgi
+  java
+}
+
+// tag::configure-jar[]
+tasks.withType<Jar> {
+    manifest {
+        // the manifest of the default jar is of type OsgiManifest
+        (manifest as? OsgiManifest)?.apply {
```

Comment:
Wow. I'm totally unhappy with that.
How should we handle such things?
@eskatos

Reply:
@StefMa Please file an issue on gradle/gradle. We may need fixes like this one: https://github.com/gradle/gradle/pull/6520